index.html

```diff
@@ -0,0 +1,250 @@
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
+            </div>
+          </div>
+        </article>
+
+        <article class="card">
+          <div class="card_img">&nbsp;</div>
+          <div class="card_details abs bottom_0">
+            <h1>
+              Multi-Post Stories <br />
+              Gain+Glory
+            </h1>
+
+            <ul class="tags">
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">Ruby on rails</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">CSS</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
+            </div>
+          </div>
+        </article>
+
+        <article class="card">
+          <div class="card_img">&nbsp;</div>
+          <div class="card_details abs bottom_0">
+            <h1>
+              Multi-Post Stories <br />
+              Gain+Glory
+            </h1>
+
+            <ul class="tags">
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">Ruby on rails</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">CSS</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
+            </div>
+          </div>
+        </article>
+
+        <article class="card">
+          <div class="card_img">&nbsp;</div>
+          <div class="card_details abs bottom_0">
+            <h1>
+              Multi-Post Stories <br />
+              Gain+Glory
+            </h1>
+
+            <ul class="tags">
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">Ruby on rails</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">CSS</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
+            </div>

```

Comment:
- [ ] The Figma template provided styling rules for hover, active (when it's pressed), and enabled (default state). Kindly implement this requirement.